cmd/main.go

```diff
@@ -198,6 +199,10 @@ func main() {
 			RetryInterval: 2 * time.Second, Namespace: Options.LeaderConfig.Namespace, RenewDeadline: 4 * time.Second},
 			"assisted-service-migration-helper",
 			log.WithField("pkg", "migrationLeader"))
+		baseISOUploadLeader = leader.NewElector(k8sClient, leader.Config{LeaseDuration: 5 * time.Second,
```

Comment:
if we use this leader only in k8 and we even don't run the function on prem maybe just move this var to the upload function itself? looks like there is no need for it in main?

Reply:
I need k8sClient which is only available here

Reply:
but u can provide it to the function and not leader, i think we have too much vars here and if we will have one more it will be better